_episodes/02-loop.md

```diff
@@ -33,7 +33,7 @@ word = 'lead'
 ~~~
 {: .language-python}
 
-In Python, a string is just an ordered collection of characters, so every
+In Python, a string is an ordered collection of characters and every
```

Comment:
I think it's helpful to indicate that it is an ordered collection as later, when we iterate over a string, list, tuple or other containers, it will make sense that it is a collection and that there's an ordering. 

I don't see how adding "basically"  changes the meaning.